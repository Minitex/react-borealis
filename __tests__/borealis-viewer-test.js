import React from 'react'
import BorealisViewer from '../src/borealis-viewer'
import renderer from 'react-test-renderer'

test('Borealis viewer', () => {
  const active_asset =
    {
      "focus": true,
      "type": "pdf",
      "src": "https://s3.amazonaws.com/mdl-assets/stpaul.pdf",
      "thumbnail": "http://reflections.mndigital.org/utils/getthumbnail/collection/p16022coll35/id/0"
    }

  const component = renderer.create(
    <BorealisViewer active_asset={active_asset} />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})