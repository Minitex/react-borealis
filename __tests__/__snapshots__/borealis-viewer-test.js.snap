exports[`test Borealis viewer 1`] = `
<div
  className="react-borealis col-md-12"
  id="react-borealis">
  <object
    data="https://s3.amazonaws.com/mdl-assets/stpaul.pdf"
    height={500}
    type="application/pdf"
    width={900}>
    <img
      src="http://reflections.mndigital.org/utils/getthumbnail/collection/p16022coll35/id/0" />
     
    <a
      href="https://s3.amazonaws.com/mdl-assets/stpaul.pdf">
      <span
        className="glyphicon glyphicon-download-alt">
        asdasdasd
      </span>
       
      https://s3.amazonaws.com/mdl-assets/stpaul.pdf
    </a>
  </object>
</div>
`;
